src/scss/components/optional/_overflow-carousel.scss

```diff
@@ -20,26 +20,61 @@
         }
     }
 
-        .c-overflowCarousel-inner {
-            @include media('<mid') {
-                overflow-x: scroll;
-                overflow-y: hidden;
-                -webkit-overflow-scrolling: touch;
-                width: 100%;
-                &.is-active {
-
-                    overflow: visible;
-                }
+    .c-overflowCarousel-inner {
+        @include media('<mid') {
+            overflow-x: scroll;
+            overflow-y: hidden;
+            -webkit-overflow-scrolling: touch;
+            width: 100%;
+            &.is-active {
+
```

Comment:
minor but could you remove the line break

Reply:
✅